tutorials/03_graphics/opengl.markdown

```diff
@@ -639,11 +639,11 @@ Now we get:
 
 ![img](past_tex_border.png)
 
-Since we're not using power of two textures, i.e. textures that are strange sizes, we can't use the classic GL_REPEAT, but that's fine, it's not really that useful anyways, honestly.
+Since we're not using the power of two textures, i.e. textures that are strange sizes, we can't use the classic GL_REPEAT, but that's fine, it's not really that useful anyway, honestly.
```

Comment:
"power of two textures" is a common way of saying. "The power of two textures" sounds weird to me. Any english native speakers out there? @ofTheo ?